api/src/test/resources/identify-error-response.json

```diff
@@ -0,0 +1,124 @@
+{
+  "stateHandle": "02tYS1NHhCPLcOpT3GByBBRHmGU63p7LGRXJx5cOvp",
+  "version": "1.0.0",
+  "expiresAt": "2020-10-30T23:47:46.000Z",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": ["create-form"],
+        "name": "identify",
+        "href": "https://foo.oktapreview.com/idp/idx/identify",
+        "method": "POST",
+        "produces": "application/ion+json; okta-version=1.0.0",
+        "value": [
+          { "name": "identifier", "label": "Username" },
+          {
+            "name": "credentials",
+            "type": "object",
+            "form": {
+              "value": [
+                { "name": "passcode", "label": "Password", "secret": true }
```

Comment:
This is only testing one branch of the authenticate method. We've got 2 more.

1. Identify first (password isn't in this response)
2. Identify first + any 1 factor

We should add responses/tests for the other branches as well.

Reply:
@sergiishamrai-okta Can we cover the above cases also as part of this PR please?

Reply:
@arvindkrishnakumar-okta Working on it

Reply:
UTs updated